install.sh

```diff
@@ -57,33 +83,42 @@ chroot_install() {
         || exit 1
     chmod +x ./alpine-chroot-install || exit 1
     ./alpine-chroot-install -d /alpine-v310 -b v3.10
-    /alpine-v310/enter-chroot install.sh
+    /alpine-v310/enter-chroot ./install.sh
+}
+
+check_os() {
+    if [ -f "/etc/alpine-release" ]; then
+        alpine_install || exit 1
```

Comment:
Perhaps instead of `|| exit 1` everywhere `set -e` on top, can be used?

And then handling cases where error is okay (usually less obvious) becomes more prominent.

Reply:
I'll try that, I vaguely remember it being unavailable in primitive shells?

Reply:
Good point!  I've never encountered issues with it being not available, that being said, while it's still not 100% clear to me, it seems `set` might be a part of posix? 

https://pubs.opengroup.org/onlinepubs/9699919799/

Unfortunately this website is outrageously terrible, and doesn't allow for direct linking, so you'll have to do some clicking on your own there 🙈.  Anyway this is mostly the relevant part:

<img width="1090" alt="MacOS 2019-09-11 at 15 25 44" src="https://user-images.githubusercontent.com/526657/64682225-57db5980-d4ab-11e9-86f0-393af8daf8e9.png">